packages/credential-eip712/src/utils/getEthTypesFromInputDoc.ts

```diff
@@ -0,0 +1,85 @@
+import { canonicalize } from "json-canonicalize";
```

Comment:
why include this file instead of just use/maintain the package?

Reply:
sounds good. we could probably make a new repo/package for it under `veramolabs` actually though, since it's currently in `sertoid`, which isn't really managed by anyone anymore. I just think it should probably exist outside of veramo itself

Reply:
switched to using external package